packages/common/src/utils.ts

```diff
@@ -38,6 +40,22 @@ export function isValidLottieJSON(json: Record<string, unknown>): boolean {
   return mandatory.every((field: string) => Object.prototype.hasOwnProperty.call(json, field));
 }
 
+export function isAudioAsset(asset: Asset.Value): boolean {
+  return !('h' in asset) && !('w' in asset) && 'p' in asset && 'e' in asset && 'u' in asset && 'id' in asset;
+}
```

Comment:
not exported from `dotlottie-js` ?

Reply:
do we need to export this function ? as no other module using it if i'm not mistaken